### ISSUE TITLE
stx_read_elut fails if current working directory is not STIX-GSW

### ISSUE THREAD

**Issue by elastufka** [i4Ds/STIX-GSW#98]
**Short Description**

file not found error for stx_read_elut() when current working directory is not STIX-GSW

**Details**

```
IDL> stx_read_elut, gain, offset, adc4096_str, ekev_a = ekev
% Compiled module: STX_READ_ELUT.
% Compiled module: FILE2TIME.
% Compiled module: EXTRACT_FID.
% Compiled module: EXTRACT_FIDS.
% Compiled module: BOX_MESSAGE.
 ----------------------------------
| Could not find FIDs in all files |
 ----------------------------------
% FILE2TIME: Expect at least YYMMDD[delimiter]HHMM...
% READ_CSV: OPENR: Null filename not allowed.
% Execution halted at: STX_READ_ELUT      99 /Users/wheatley/Documents/Solar/STIX/STIX-GSW/stix/idl/processing/drm
  /stx_read_elut.pro
%                      $MAIN$       

IDL> cd, '../STIX-GSW'
IDL> stx_read_elut, gain, offset, adc4096_str, ekev_a = ekev
% Compiled module: STRMIDS.
% Compiled module: INT2UTC.
% Compiled module: MJD2DATE.
% Compiled module: UTC2STR.
% Compiled module: REPRODUCE.
```

**Environment**
- IDL Version: [8.4]
- STIX GSW Version: [version, use `stx_gsw_version` <- doesn't work (or I forgot IDL, very possible) but I did git pull a few weeks ago] 
- OS: [Mac]

**Comment by grazwegian** [comment]
Updated version committed need to be tested: https://github.com/grazwegian/STIX-GSW/commit/1b6741584436d14643cd4beff5be4b4d2b65b03a

**Comment by samaloney** [comment]
Yea that seems to work the ideal process here would be to open a PR with just that change but we can prob skip in this case

**Comment by grazwegian** [comment]
fixed in #105
